Formula/shyaml.rb

```diff
@@ -0,0 +1,31 @@
+class Shyaml < Formula
+  include Language::Python::Virtualenv
+
+  desc "Command-line YAML parser"
+  homepage "https://github.com/0k/shyaml"
+  url "https://files.pythonhosted.org/packages/67/70/1133a5817bc62ff4e7ceee59edb95d127092db9385cc7cda5fcac93c494a/shyaml-0.4.1.tar.gz"
+  sha256 "a1535c25bf0058563e03ea8cbad8c4dc755ed231e6a9f3f584982994f19eae59"
+
+  depends_on :python3
+
+  resource "PyYAML" do
+    url "https://files.pythonhosted.org/packages/4a/85/db5a2df477072b2902b0eb892feb37d88ac635d36245a72a6a69b23b383a/PyYAML-3.12.tar.gz"
+    sha256 "592766c6303207a20efc445587778322d7f73b161bd994f227adaa341ba212ab"
+  end
+
+  def install
+    virtualenv_install_with_resources
```

Comment:
You're right, actually got the Python 2 version. :facepalm: (really need this emoji)